spec/14_find_number_spec.rb

```diff
@@ -4,29 +4,29 @@
 
 # rubocop:disable Layout/LineLength, Metrics/BlockLength
 
-# Let's take a look at test-driven development (TDD) technique called mocking.
-# Mocking uses a 'double', which is a generic ruby object.
-# Doubles are strict, which means you must specify (allow) any messages that it can receive.
-# If the double receives a message that has not been allowed, it will trigger an error.
+# Let's take a look at test-driven development (TDD) technique of using a 'double'.
+# A 'double' is a generic ruby object, that stands in for the real object, like a stunt-double.
 
-# Doubles are useful in TDD because you can create test functionality that is not coded yet.
+# Doubles are very useful in TDD because you can create test functionality that is not coded yet.
 
-# Doubles can be a confusing concept, so here are several resources to check out:
-# https://www.tutorialspoint.com/rspec/rspec_test_doubles.htm
-# https://relishapp.com/rspec/rspec-mocks/v/3-9/docs/basics/test-doubles
-# http://testing-for-beginners.rubymonstas.org/test_doubles.html
-
-# In this file, we will be testing the 'FindNumber' class.
+# In this example, we will testing the class 'FindNumber'. Look at the lib/14_find_number.rb file.
```

Comment:
Would read better as "we will **be** testing the ..."

Reply:
Yes, I will make this change. Thank you for reading through these changes to catch anything like this.